cmd/srcd/cmd/sql.go

```diff
@@ -121,24 +126,39 @@ func repl() error {
 			return nil
 		case "":
 		default:
-			if err := runQuery(statement); err != nil {
+			if err := runQuery(client, statement); err != nil {
 				fmt.Println(err)
 			}
 		}
 	}
 }
 
-func runQuery(query string) error {
-	c, err := daemon.Client()
-	if err != nil {
-		return fmt.Errorf("could not get daemon client: %v", err)
+func ensureConnReady(client api.EngineClient) error {
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+	defer cancel()
+
+	sleep := 1
+	for {
+		_, err := client.SQL(ctx, &api.SQLRequest{Query: "SELECT 1"})
+		if err == nil {
+			logrus.Info("SQL connection ready")
+			break
+		}
+
+		logrus.Warningf("SQL connection not ready, trying in %d seconds", sleep)
```

Comment:
Fernanda just had an error due to lack of ping (she is running gitbase on a lot of repos).
This PR fixes it. But maybe we can write more user-friendly message like: "gitbase is starting, please wait" or something like that.